x-pack/solutions/observability/plugins/observability_onboarding/public/application/onboarding_flow_form/onboarding_flow_form.tsx

```diff
@@ -325,7 +325,7 @@ export const OnboardingFlowForm: FunctionComponent = () => {
             </strong>
           </EuiTitle>
           <EuiSpacer size="m" />
-          <PackageList list={featuredCardsForCategory} />
+          <PackageList list={featuredCardsForCategory} showCardLabels={true} />
```

Comment:
Why do we need `showCardLabels` here? Seems like we can simplify even more and remove them from here

Reply:
Ah, makes sense